### ISSUE TITLE
How to know other node's text information? 

### ISSUE THREAD

**Issue by jw9603** [mlvlab/QAT#2]
Hello,
Thank you for providing such an excellent paper.
I am a student with a lot of interest in this field. I know that the OpenBookQA dataset has 4 correct labels per question, and accordingly, 4 subgraphs occur.
Here, each subgraph has 200 nodes.
There are three types of nodes: context, question, answer, and other nodes. What is the way to know the text information of other nodes?
Let me give you an example.
The question is:
Question:
“A person wants to start saving money so that they can afford a nice vacation at the end of the year. After looking over their budget and expenses, they decide the best way to save money is to”

Answer:
A. make more phone calls
B. quit eating lunch out
C. buy less with monopoly money
D. have lunch with friends

In the above problem
"qc" : ["afford", "best", "best_way", "budget", "decide", "end", "expense", "expenses", "good", "look", "looking", " money", "nice", "nice_vacation", "person", "person_wants", "save", "save_money", "saving", "saving_money", "start", "us_budget", "vacation", "way" , "year"],
"ac": ["call", "calls", "phone", "phone_call", "phone_calls"]}
It is configured like this, and the context node is known as the node connecting the question and answer nodes.
What I'm curious about in this situation is how to check(know) which words the other nodes are embedded with(text information).
 Thanks for reading the long question. have a good day.

**Comment by jw9603** [comment]
Thank you for your reply.
Since the knowledge graph is too vast, I think it is difficult to grasp the text information of the Other node belonging to the subgraph for each problem.
It's not listed here in the repository, but if you look at the data/cpnet/conceptnet.en.csv file in MHGRN(https://github.com/INK-USC/MHGRN) which is previous paper, 
<img width="964" alt="image" src="https://user-images.githubusercontent.com/70795645/228615434-db756f6b-d6ee-4f9c-aaa9-09bd5ecf1dba.png">
you can see the relation scores between words. Then, should the text information of other nodes be identified based on the above file?

Thank you!

**Comment by PJin0** [comment]
Yes, the text information of other nodes can be obtained through the mentioned file like MHGRN or QAGNN.
